src/components/DetailedProduct.js

```diff
@@ -1,15 +1,14 @@
 import React from "react";
 import { connect } from "react-redux";
-
-import store from "../store";
 import { getDetailProduct } from "../storeReducers/productsReducer";
+import CreateReview from "../components/CreateReview";
 
 import Reviews from "./Reviews";
 
 // More to come for this thing, need reviews, and add to cart
 // Basics are here!!!
-function DetailProduct({ detailProduct, id }) {
-  //  console.log(detailProduct);
+function DetailProduct({ detailProduct, id, user }) {
+  console.log("DETAILED PRODUCT", user);
   if (detailProduct.id !== id) getDetailProduct(id);
```

Comment:
I don't think we need this console.log() but I think we leave it for sanity at the moment